reference/configuration/tidb-server/configuration-file.md

```diff
@@ -254,6 +254,12 @@ Configuration items related to performance.
 - Default value: `5000`
 - If a transaction does not roll back or commit after the number of statements exceeds `stmt-count-limit`, TiDB returns the `statement count 5001 exceeds the transaction limitation, autocommit = false` error. This configuration takes effect **only** in the retriable optimistic transaction. If you use the pessimistic transaction or have disabled the transaction retry, the number of statements in a transaction is not limited by this configuration.
 
+### `txn-total-size-limit`
+
+- The size limit of a transaction.
+- Default value: `104857600`
+- The total key-value entries size in bytes should be less than this value. Note, if the `binlog` is enabled, this value should be less than 104857600(100M), because of the limitation of the binlog component; If `binlog` is not enabled, the maximum value of this configuration is 10737418240(10G).
```

Comment:
Should we set a lower limit, if the size is too low, TiDB will not start? This comment may not be related to the documentation.

Reply:
Not necessary. Our customers are sane enough.